app/src/ingest_guru_cards.py

```diff
@@ -51,18 +54,21 @@ def _ingest_cards(
 
 
 def main() -> None:
-    if len(sys.argv) < 2:
-        logger.warning("No argument for filepath.")
+    if len(sys.argv) < 3:
+        logger.warning("Expecting 2 arguments: dataset_id and filepath. Got %s", sys.argv)
         return
 
-    guru_cards_filepath = sys.argv[1]
+    dataset_id = sys.argv[1]
+    guru_cards_filepath = sys.argv[2]
 
-    logger.info(f"Processing Guru cards at {guru_cards_filepath}")
+    logger.info(f"Processing Guru cards {dataset_id} at {guru_cards_filepath}")
 
     embedding_model = get_embedding_model()
 
+    # TODO: Add `region` as a argv element or have a better way to set document attributes
+    doc_attribs: dict[str, str | None] = {"program": dataset_id, "region": "Michigan"}
```

Comment:
@KevinJBoyer  Should `dataset_id` be renamed to `program_id`:
```suggestion
    # TODO: Add `region` as a argv element or have a better way to set document attributes
    doc_attribs: dict[str, str | None] = {"program": program_id, "region": "Michigan"}
```
or is there a mapping from `dataset_id` to `program` and `region`?

Reply:
Looks like this may be out of date with your latest change, but I agree with using `program`.

Reply:
That makes a ton of sense to me, I was seeing the same need but called it `source` rather than dataset in the feedback for policy IDs (I don't have any opinion about the naming here both make sense).

My preference would be try to keep as much of this outside of the code as possible (i.e., not have the mapping of dataset -> program & region hardcoded, but instead make users provide each) to try to separate the data contents (various programs, regions, and dataset names) from the code itself as much as possible-- that keeps the code as flexible and general/generic. E.g., it would save to make any code changes or redeploy the application to ingest a new data set.

Reply:
I was trying to avoid having a long ingest command but it's acceptable for now. It will look something like:
```sh
make ingest-guru-cards DATASET_ID=policy-mi PROGRAM=SNAP REGION=Michigan FILEPATH=/app/bem-1.pdf
```